tensorrt_llm/tools/multimodal_builder.py

```
@@ -1540,6 +1614,210 @@ def forward(self, images, rotary_pos_emb, attention_mask):
+                                                 dim=-1,
+                                                 dtype=torch.float32).to(
+                                                     q.dtype)
+            attn_output = torch.matmul(attn_weights, v)
+            attn_output = attn_output.transpose(0, 1)
+            attn_output = attn_output.reshape(seq_length, -1)
+            attn_output = self.proj(attn_output)
+            return attn_output
+
+    class Qwen2_5_VLVisionBlockOpt(Qwen2_5_VLVisionBlock):
+
+        def __init__(self, config, attn_implementation: str = "eager") -> None:
```

Ruff (0.12.2)

Unused method argument: `attn_implementation`

(ARG002)